web_disable_export_group/readme/CREDITS.rst

```diff
@@ -0,0 +1,2 @@
+Original code from module web_disable_export by Noviat,
```

Comment:
After the last adopted solution, the .js code is completely changed.
Not sure we want to keep this mentioning in the credits.
What do you think?

Reply:
I've got no special problem in dropping it :slightly_smiling_face: I don't know if there should be an historic mention elsewhere